### ISSUE TITLE
A more graceful result for a failed request.

### ISSUE THREAD

**Issue by raffie-** [m-wrzr/populartimes#46]
I use populartimes.getid() and periodically pull the populartimes data using a list of place_ids I have saved. At times, a place_id may get depreciated. This will cause a problem because as long as either the API KEY or the place id cannot be recognized by google, the API call will breakdown and the following error msg printed:
_ERROR:root:The query string is malformed, check params.json if your formatting for lat/lng and radius is correct.
Kernel died, restarting_
Reproducible example:
populartimes.get_id(API_KEY1,"hoho")

Is there a way to make the function fail more gracefully? For example, just output an empty list?

**Comment by Moyaga** [comment]
Try just printing `e`

**Comment by m-wrzr** [comment]
@raffie- you are using an old version of populartimes, the error message you posted (below) isn't available anymore since commit a931eb79850bd8a78cacd01fbf9c8d6cbe6fd396. You should update to the current version and than catch errors as @Moyaga suggested.

```
ERROR:root:The query string is malformed, check params.json if your formatting for lat/lng and radius is correct.
```

**Comment by raffie-** [comment]
Okay I reinstalled the package and it works now! thanks!
